### PR TITLE
Enable Render test CI

### PR DESCRIPTION
**Pull request by alanchenboy**
Enable Render test CI

### CODE REVIEW

File: render-test/runner.cpp [hunk 1]
```diff
@@ -629,7 +629,7 @@ TestOperations getAfterOperations(const Manifest& manifest) {
         }
         if (networkProbeOp == probe) {
             result.emplace_back([](TestContext& ctx) {
-                assert(ProxyFileSource::isTrackingActive());
+//                assert(ProxyFileSource::isTrackingActive());
```
Comment (louwers) [inline render-test/runner.cpp:632]:
```suggestion
```

File: render-test/parser.cpp [hunk 2]
```diff
@@ -1018,7 +1024,7 @@ TestOperations parseTestOperations(TestMetadata& metadata) {
         } else if (operationArray[0].GetString() == networkProbeEndOp) {
             // probeNetworkEnd
             result.emplace_back([](TestContext&) {
-                assert(mbgl::ProxyFileSource::isTrackingActive());
+//                assert(mbgl::ProxyFileSource::isTrackingActive());
```
Comment (louwers) [inline render-test/parser.cpp:1027]:
```suggestion
```

File: render-test/parser.cpp [hunk 3]
```diff
@@ -1007,7 +1013,7 @@ TestOperations parseTestOperations(TestMetadata& metadata) {
             assert(operationArray[1].IsString());
             std::string mark = std::string(operationArray[1].GetString(), operationArray[1].GetStringLength());
             result.emplace_back([mark](TestContext& ctx) {
-                assert(mbgl::ProxyFileSource::isTrackingActive());
+//                assert(mbgl::ProxyFileSource::isTrackingActive());
```
Comment (louwers) [inline render-test/parser.cpp:1016]:
```suggestion
```

File: render-test/parser.cpp [hunk 4]
```diff
@@ -997,7 +1003,7 @@ TestOperations parseTestOperations(TestMetadata& metadata) {
         } else if (operationArray[0].GetString() == networkProbeStartOp) {
             // probeNetworkStart
             result.emplace_back([](TestContext&) {
-                assert(!mbgl::ProxyFileSource::isTrackingActive());
+                //assert(!mbgl::ProxyFileSource::isTrackingActive());
```
Comment (louwers) [inline render-test/parser.cpp:1006]:
```suggestion
```

**Review by louwers — CHANGES_REQUESTED** [review]
- The commented out code should be removed.
- Could you document somewhere what `ignoreProbing` does? Maybe as a comment.
- Do you think the name of this workflow (node-ci) accurately reflects what it does? Because the render test that runs on macOS has little to do with Node.js, right?

File: render-test/runner.cpp (hunk 1 above)
Comment (alanchenboy) [reply render-test/runner.cpp]: Thanks for review，the comment is deleted

**Comment by alanchenboy** [comment]
@louwers 
The commented out code was removed.
ignoreProbing has added a comment.
Render tests don't belong to the node.js, maybe we can move it to macOS-ci, right?

**Review by louwers — APPROVED** [review]
I am not 100% clear about the specifics of these render tests, but it looks good to me.

Let's move the render test to its own workflow in a separate PR.

**Pull request merged**